packages/react/src/components/Combobox/Combobox.tsx

```diff
@@ -434,16 +636,54 @@ const Combobox = forwardRef<HTMLDivElement, ComboboxProps>(
         </label>
         {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */}
         <TextFieldWrapper
-          className={classnames({ 'TextFieldWrapper--error': hasError })}
+          className={classnames({
+            'Combobox__TextFieldWrapper--multiselect': multiselect,
```

Comment:
The naming convention on this class... is a little odd. It implies that TextFieldWrapper has a bem modifier of multiselect, which really belongs to Combobox.

My recommendation here would be instead to add the modifier to combobox and have a style cascade. Example:

```tsx
<div
  id={id}
  className={classnames('Combobox', className, { 'Combobox--multiselect': multiselect })}
  ref={comboboxRef}
>
  ...
</div>
```

Then in your styles you would just target the selector cascade:

```css
.Combobox--multiselect .TextFieldWrapper {
  ...
}
```

Reply:
Updated